sorting/quick_sort.cpp

```diff
@@ -21,7 +21,7 @@
  *      x. All this should be done in linear time
  *
  * @author [David Leal](https://github.com/Panquesito7)
- * @author Unknown author
+ * @author [popoapp](https://github.com/popoapp)
```

Comment:
I don't think that's the author - this person joined GitHub very [recently](https://github.com/popoapp?tab=overview&from=2022-05-01&to=2022-05-31)...

Reply:
https://github.com/TheAlgorithms/C-Plus-Plus/commits/1c1746bc6496d568d572143cc5565dd3f6f884ee/Quick%20Sort.cpp?browsing_rename_history=true&new_path=sorting/Quick%20Sort.cpp&original_branch=f8da1b06852e33cb999b3df3fda8326b01500e90

Reply:
Very weird indeed 😅

Reply:
Though this seems legitimate - this person may have created a commit in the "git" repository (even though they didn't have an account on "GitHub") but this commit was pushed to "GitHub" by some other person who did have an account with write access. I guess this is related to the early days of `TheAlgorithms` organisation. I suspect that it wasn't an organisation and was turned into one after Anup Kumar Panwar already had a repository. So I presume that the author name is correct 😄